<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wgt_QEditVideo</class>
 <widget class="QWidget" name="wgt_QEditVideo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">EditVideoWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="wgt_QVideoPlayer" name="VideoPlayer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Start :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="SeekLeftBt">
       <property name="toolTip">
        <string extracomment="Tooltip in dialog box">Move to start position</string>
       </property>
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="icon">
        <iconset resource="../../RSCffDiaporama.qrc">
         <normaloff>:/img/player_rew.png</normaloff>:/img/player_rew.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="DefStartPosBT">
       <property name="toolTip">
        <string extracomment="Tooltip in dialog box">Use current position as start position</string>
       </property>
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="icon">
        <iconset resource="../../RSCffDiaporama.qrc">
         <normaloff>:/img/PosStart.png</normaloff>:/img/PosStart.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="StartPosEd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string extracomment="Tooltip in dialog box">Define the start position to use</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="specialValueText">
        <string notr="true"/>
       </property>
       <property name="displayFormat">
        <string notr="true">HH:mm:ss.zzz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>End :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="EndPosEd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string extracomment="Tooltip in dialog box">Define the end position to use</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="specialValueText">
        <string notr="true"/>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::HourSection</enum>
       </property>
       <property name="displayFormat">
        <string notr="true">HH:mm:ss.zzz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="DefEndPosBT">
       <property name="toolTip">
        <string extracomment="Tooltip in dialog box">Use current position as end position</string>
       </property>
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="icon">
        <iconset resource="../../RSCffDiaporama.qrc">
         <normaloff>:/img/PosEnd.png</normaloff>:/img/PosEnd.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="SeekRightBt">
       <property name="toolTip">
        <string extracomment="Tooltip in dialog box">Move to end position</string>
       </property>
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="icon">
        <iconset resource="../../RSCffDiaporama.qrc">
         <normaloff>:/img/player_fwd.png</normaloff>:/img/player_fwd.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_21">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Duration :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ActualDuration">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="text">
        <string notr="true">00:00:00.000</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_22">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Volume :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="VolumeReductionFactorCB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Select the &quot; Master Volume &quot; for the movie</string>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="DeinterlaceBt">
       <property name="toolTip">
        <string>Allows you to activate the deinterlace filter</string>
       </property>
       <property name="text">
        <string>Deinterlace video</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wgt_QVideoPlayer</class>
   <extends>QWidget</extends>
   <header>wgt_QVideoPlayer/wgt_QVideoPlayer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../RSCffDiaporama.qrc"/>
 </resources>
 <connections/>
</ui>
